Users/Install.rst

```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python 3 and Python 2 installed, use
```

Comment:
make`python 3` to `python3` and similarly to python 2

Reply:
@newbazz are you referring to case also??

Reply:
nope ;P my bad